frontend/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,188 @@
+import { useController, useForm } from 'react-hook-form'
+import {
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+} from '@chakra-ui/form-control'
+import { CheckboxGroup, CheckboxProps, Input, VStack } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+import { omit } from 'lodash'
+
+import Button from '~components/Button'
+import Others from '~components/Others'
+
+import { Checkbox } from './Checkbox'
+
+export default {
+  title: 'Components/Checkbox',
+  component: Checkbox,
+  decorators: [],
+} as Meta
+
+export const Default: Story<CheckboxProps> = (args) => <Checkbox {...args} />
+Default.args = {
+  value: 'Option',
+}
+
+export const Group: Story<CheckboxProps> = (args) => {
+  return (
+    <VStack align="left">
+      <Checkbox value="Option 1" />
+      <Checkbox value="Option 2" />
+      <Checkbox value="Option 3" />
+      <Others {...args} value="Others" base="checkbox">
```

Comment:
I would actually prefer duplicating code instead of having the `base` parameter so the code remains isolated and we know that it works for that specific field

Reply:
Yep I agree. If for some reason we want to include the "Other" option for any other fields as well, this `Other` component may also not work well for them as the Chakra UI implementation of different base component may be different too. Will change!